excel.py

```diff
@@ -0,0 +1,56 @@
+#! python3
+# Reads an excel file and transforms to JSON.
+
+import xlrd
+from collections import OrderedDict
+import simplejson as json
+import psycopg2
+import urllib.parse
+
+# set up the connection. Extract the database info from the URI and connect with psycopg2
+def connect():
+    try:
+        result = urllib.parse.urlparse("postgres://4cebab44:4j602JWnaP3Yaw@db.nuodata.io:5432/4cebab44")
```

Comment:
NEVER put authentication information in a git history. 

Easiest is to put it into a separate user-created JSON file and use built-in JSON parsing support.

```
rngadam@rngadam-mac ~/c/s/nuodata-example> cat config.json 
{
    "username": "USERNAME",
    "password": "PASSWORD"
}
rngadam@rngadam-mac ~/c/s/nuodata-example> python
>>> import json
>>> c = json.load(open('config.json'))
>>> c['username']
u'USERNAME'
>>> c['password']
u'PASSWORD'
```

Make sure to put config.json in .gitignore and create a config.json.example.